spec.emu

```diff
@@ -390,6 +390,34 @@ contributors: Google, Ecma International
     </emu-note>
   </emu-clause>
 
+  <emu-clause id="sup-temporal-calendardateextendedyear" type="abstract operation">
+    <h1>
+      CalendarDateExtendedYear (
+        _calendar_: a String,
+        _date_: a Temporal.PlainDateTime, Temporal.PlainDate, or Temporal.PlainYearMonth,
+      ): either a normal completion containing an integer or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by _calendar_ relative to a well-defined epoch year for that calendar.</dd>
+    </dl>
+    <p>
+        This definition supersedes the definition provided in "Temporal proposal", #sec-temporal-calendardateextendedyear.
+    </p>
+    <emu-alg>
+      1. Let _year_ be ? Get(_date_, *"year"*).
+      1. If IsCalendarSupportEra(_calendar_) is *false*, then
+        1. Return _year_.
+      1. If _calendar_ is *"ethioaa"*, then
+        1. Set _epochEra_ to *"ethiopic-amete-alem"*.
+      1. Else,
+        1. Let _epochEra_ be _calendar_.
+      1. Let _extendedYear_ be the integer to indicate the year corresponding to _year_ in the context of the calendar represented by _calendar_ relative to the _epochEra_ era from implementation-defined processing.
```

Comment:
I don't think it works to define it that way. I think Shane and I discussed creating another column in Table 1 to indicate the ISO year corresponding to extended year 1, and I think that's the way to go.

Reply:
but wouldn't that be per calendar and not per era? Table 1 has one row per era.